### PR TITLE
Bug fixes + translations update

### PR DESCRIPTION
**Pull request by markdoten**
1. Fixing bug where `stylePrefix` was passed incorrectly to the header view
2. Bumping `app-base` for translation and theme fixes
3. Bumping `streamhub-sdk` for translation fixes

### CODE REVIEW

**Comment by markdoten** [comment]
#nojira

File: bower.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+    "app-base": "git@github.com:Livefyre/app-base#1.0.3",
     "cajon": "git://github.com/requirejs/cajon.git#~0.1.11",
-    "streamhub-input": "Livefyre/streamhub-input#0.9.0",
-    "rework": "gobengo/rework#~0.21.0",
     "livefyre-theme-styler": "git://github.com/livefyre/livefyre-theme-styler.git#1.1.0",
-    "node-tiny-uuid": "git://github.com/RiveraGroup/node-tiny-uuid.git#ea7f741c9ad912d8ad2201b25fe9f5d945206386"
+    "node-tiny-uuid": "git://github.com/RiveraGroup/node-tiny-uuid.git#ea7f741c9ad912d8ad2201b25fe9f5d945206386",
+    "rework": "gobengo/rework#~0.21.0",
+    "streamhub-input": "Livefyre/streamhub-input#0.9.0",
+    "streamhub-sdk": "git://github.com/Livefyre/streamhub-sdk.git#2.23.2"
```
Comment (ktabors) [inline bower.json]: alphabetizing and changing versions on app base and sdk, right?

Comment (markdoten) [inline bower.json]: Yeah

File: src/wall-component.js [hunk 2]
```diff
@@ -89,14 +89,20 @@ var THEMABLE_STYLES = {
...
+/** @override */
 WallComponent.prototype.configureInternal = function (configOpts) {
   var reconstructWallView = false;
   var reconstructHeaderView = false;
   var newCollection;
   var needRender = false;
   var needCollectionPipeToWallView = false;
 
-  if (configOpts.linkColor !== this._themeOpts.linkColor) {
+  if (this._themeChanged) {
```
Comment (ktabors) [inline src/wall-component.js]: where is _themeChanged and generateStylePrefix() defined? I don't see them in this class or in your sdk changes?

Comment (markdoten) [inline src/wall-component.js]: Those are in app-base

**Comment by kwalter37** [comment]
lgtm

**Pull request merged**